### DISCUSSION TITLE
Cluster for know compound not found

### DISCUSSION THREAD

**Discussion by shaheenbb656** [antismash/antismash#583] [Q&A]
Hello,
I have assembled genomes of bacteria, where we know that a compound we are looking for is present as we have proved that in lab. However running these genomes turns only the initiation domain and doesn't show if elongation and termination domain. I have also tried to find the cluster for this toxin uploading publicly available genomes that turns out same results.
Can someone please explain as I am confused if we know the toxin (NRPS) is produced then why does antiSMASH detect only the first module.
Thanks

**Comment by SJShaw** [comment]
There are many possibilities and it's difficult to know without more detail.

Some possible reasons: your input(s) might be fragmented right at that NRPS, gene calling may not be accurate around that NRPS, it might be some novel/alternative biosynthesis making the same product, etc.
